root/etc/cont-init.d/10-adduser.sh

```diff
@@ -25,8 +25,8 @@ set -e
 PUID=${PUID:-1001}
 PGID=${PGID:-1001}
 
-addgroup -g "$PGID" abc
-adduser -u "$PUID" -D -G abc abc
+getent group abc >/dev/null || addgroup -g "$PGID" abc
+id -u abc &>/dev/null || adduser -u "$PUID" -D -G abc abc
```

Comment:
I got the ideas from these Stack Exchange answers respectively.
https://unix.stackexchange.com/a/145351/21121
https://unix.stackexchange.com/a/28530/21121